include/merlin_hashtable.cuh

```diff
@@ -232,9 +238,19 @@ class HashTable {
    * the evict strategy. If false, it requires the metas follow the evict
    * strategy of table.
    */
+  inline void insert_or_assign(size_type n,
+                               const key_type* keys,              // (n)
+                               const value_type* values,          // (n, DIM)
+                               const meta_type* metas = nullptr,  // (n)
+                               cudaStream_t stream = 0,
+                               bool ignore_evict_strategy = false) {
+    insert_or_assign(n, keys, reinterpret_cast<const vector_type*>(values),
+                     metas, stream, ignore_evict_strategy);
+  }
+
   void insert_or_assign(size_type n,
-                        const key_type* keys,              // (n)
-                        const value_type* values,          // (n, DIM)
+                        const key_type* keys,       // (n)
+                        const vector_type* values,  // (n), each DIM-sized
```

Comment:
I think the previous `(n, DIM)` is following the style of shape description in TF& PyTorch, so we'd better to keep it.

Reply:
Both function signatures are there.

Reply:
You need to check the code more carefully. There is an inline alias with the other pointer type. And the compiler will most likely inline it because it is a super small function. So, there is no overhead added.

Reply:
See above: https://github.com/NVIDIA-Merlin/HierarchicalKV/pull/58#discussion_r1016176950